keylime/cloud_verifier_tornado.py

```diff
@@ -666,6 +666,11 @@ def main(argv=sys.argv):
     config = common.get_config()
     cloudverifier_port = config.get('cloud_verifier', 'cloudverifier_port')
 
+    # tornado defaults to 100MB max upload size, but make it configurable
+    max_upload_size = 104857600
```

Comment:
I guess I am nitpicking here, but I would not bother to set the value for `max_upload_size` in the code. We now have it set in the keylime.conf  (I am good with 100mb) so declare it and not bother checking if its present or not. Just have:

cloud_verifier_common.py: `max_upload_size = config.get('cloud_verifier', 'max_upload_size')`

keylime.conf: `max_upload_size = 104857600`

Hope that makes sense, the rest looks great!

Reply:
In fact it would be good to set the config parse declaration (max_upload_size = config.get....) at the top of the file between these lines somewhere (I guess 39 perhaps?)

https://github.com/keylime/keylime/blob/30b196de45999c89d25490cd00fe88cbd42785f2/keylime/cloud_verifier_tornado.py#L38-L91

Reply:
> In fact it would be good to set the config parse declaration (max_upload_size = config.get....) at the top of the file between these lines somewhere (I guess 39 perhaps?)

I saw this more like the other configuration parameters that control the web server (like cloudverifier_port or multiprocessing_pool_num_workers) and thus put it closer to the code that uses it. If it were used in multiple places, then yes I would agree.